tests/store/test_store_api.py

```diff
@@ -12,10 +16,10 @@ def test_fetch_bundles(populated_store, bundle_data_old):
     store.add_commit(bundle_old_obj)
 
     # WHEN fetching all files in the database
-    query = store.files_before(before="2020-05-04")
+    files = store.get_files_before(before_date=time_stamp_now)
 
     # THEN all four files should be fetched
-    assert len(query.all()) == 4
+    assert len(files) == 4
 
 
 def test_fetch_past_files(populated_store, bundle_data_old, timestamp, old_timestamp):
```

Comment:
Typehints and suggestion

```suggestion
def test_get_past_files(populated_store, bundle_data_old, timestamp, old_timestamp):
```